plasmapy/physics/tests/test_dielectric.py

```diff
@@ -46,12 +48,23 @@ def test_proton_electron_plasma(self):
         P_analytical = 1 - (omega_pe ** 2 + omega_pp ** 2) / omega ** 2
 
         species = ['e', 'p']
-        S, D, P = cold_plasma_permittivity_SDP(B, species, n, omega)
+        S, D, P = tuple_result = cold_plasma_permittivity_SDP(B, species, n, omega)
```

Comment:
Whoa!  I forgot that we could do chained assignments.  Makes me appreciate Python all over again. :cat: